src/Navigation/Notifications/Factory/Notification.php

```diff
@@ -43,6 +48,8 @@ class Notification extends BaseFactory implements ICanCreateFromTableRow
 	private $l10n;
 	/** @var LocalRelationship */
 	private $localRelationshipRepo;
+	/** @var ICanCache */
+	private $cache;
 
 	public function __construct(\Friendica\App\BaseURL $baseUrl, \Friendica\Core\L10n $l10n, \Friendica\Contact\LocalRelationship\Repository\LocalRelationship $localRelationshipRepo, LoggerInterface $logger)
```

Comment:
```suggestion
	public function __construct(\Friendica\App\BaseURL $baseUrl, \Friendica\Core\L10n $l10n, \Friendica\Contact\LocalRelationship\Repository\LocalRelationship $localRelationshipRepo, LoggerInterface $logger, ICanCache $cache)
```

Reply:
I tried this - but then no new notifications had been created. The function is called from several locations and it seems as if it stops working from at least one of the locations when I add the parameter there.